acceptance/tests/pxp-module-puppet/run_puppet_nonASCII.rb

```diff
@@ -0,0 +1,61 @@
+require 'pxp-agent/test_helper.rb'
+require 'puppet/acceptance/environment_utils'
+
+test_name 'C98107 - Run puppet with non-ASCII characters in Puppet code' do
+  extend Puppet::Acceptance::EnvironmentUtils
+
+  env_name = test_file_name = File.basename(__FILE__, '.*')
+  ENVIRONMENT_NAME = mk_tmp_environment(env_name)
+
+  step 'On master, create a new environment that includes a non-ASCII character and will result in Changed' do
+    environmentpath = master.puppet['environmentpath']
+    site_manifest = "#{environmentpath}/#{ENVIRONMENT_NAME}/manifests/site.pp"
+    on(master, "cp -r #{environmentpath}/production #{environmentpath}/#{ENVIRONMENT_NAME}")
```

Comment:
no longer needed.  the `#mk_tmp_environment_on` links the tmp environment into the environmentpath (but not into production)